components/compositing/compositor.rs

```diff
@@ -2220,7 +2227,9 @@ impl<Window: WindowMethods> IOCompositor<Window> {
     fn initialize_compositing(&mut self) {
         if self.webrender.is_none() {
             let show_debug_borders = opts::get().show_debug_borders;
-            self.context = Some(rendergl::RenderContext::new(self.native_display.clone(),
+            // We can unwrap() native_display because it's only None when using webrender.
```

Comment:
heh, CI doesn't like this comment, just make it `unwrap` instead of `unwrap()`

Run etc/ci/check_no_unwrap.sh to verify it works, and squash the commits. Feel free to `@bors-servo r=Manishearth` after that.